CHANGELOG.md

```diff
@@ -43,7 +43,7 @@
   * When converting to raw strings, do not change \r\n sequences if not explicitly requested by the user (PR: [#76120](https://github.com/dotnet/roslyn/pull/76120))
   * Exclude starting symbol when going to implementations if we find implementations (PR: [#76125](https://github.com/dotnet/roslyn/pull/76125))
 * Add option to disable auto insertion of documentation comments (PR: [#7381](https://github.com/dotnet/vscode-csharp/pull/7381))
-* Bump xamltools to 17.13.35606.23 (PR: [#7856](https://github.com/dotnet/vscode-csharp/pull/7856))
+* Bump xamltools to 17.13.35611.21 (PR: [#7872](https://github.com/dotnet/vscode-csharp/pull/7872))
```

Comment:
We have already snapped for the 2.60 prerelease. Can you revert this line and add a new one to the 2.61.x section?

Reply:
Updated!